packages/plugin-assets-retry/src/runtime/asyncChunkRetry.ts

```diff
@@ -164,30 +172,50 @@ function nextRetry(chunkId: string): Retry {
 const originalEnsureChunk = __RUNTIME_GLOBALS_ENSURE_CHUNK__;
 const originalGetChunkScriptFilename =
   __RUNTIME_GLOBALS_GET_CHUNK_SCRIPT_FILENAME__;
+const originalGetCssFilename =
+  __RUNTIME_GLOBALS_GET_MINI_CSS_EXTRACT_FILENAME__ ??
+  __RUNTIME_GLOBALS_GET_CSS_FILENAME__ ??
+  (() => null);
 const originalLoadScript = __RUNTIME_GLOBALS_LOAD_SCRIPT__;
 
 // if users want to support es5, add Promise polyfill first https://github.com/webpack/webpack/issues/12877
 function ensureChunk(chunkId: string): Promise<unknown> {
   const result = originalEnsureChunk(chunkId);
   const originalScriptFilename = originalGetChunkScriptFilename(chunkId);
+  const originalCssFilename = originalGetCssFilename(chunkId);
 
-  // mark the async chunk name in the global variables and share it with initial chunk retry
-  if (
-    typeof window !== 'undefined' &&
-    !window.__RB_ASYNC_CHUNKS__[originalScriptFilename]
-  ) {
-    window.__RB_ASYNC_CHUNKS__[originalScriptFilename] = true;
+  // mark the async chunk name in the global variables and share it with initial chunk retry to avoid duplicate retrying
+  if (typeof window !== 'undefined') {
+    if (
+      originalScriptFilename &&
+      !window.__RB_ASYNC_CHUNKS__[originalScriptFilename]
+    ) {
+      window.__RB_ASYNC_CHUNKS__[originalScriptFilename] = true;
+    }
+    if (
+      originalCssFilename &&
+      !window.__RB_ASYNC_CHUNKS__[originalCssFilename]
+    ) {
+      window.__RB_ASYNC_CHUNKS__[originalCssFilename] = true;
+    }
   }
 
-  return result.catch(function (error: Error) {
+  return result.catch((error: Error) => {
     const { existRetryTimes, originalSrcUrl, nextRetryUrl, nextDomain } =
       nextRetry(chunkId);
 
+    // At present, we don't consider the switching domain and addQuery of async css chunk
+    // 1. Async js chunk will be requested first. It is rare for async css chunk to fail alone.
+    // 2. the code of loading css in webpack runtime is complex and it may be modified by cssExtractPlugin, increase the complexity of this plugin.
+    const isCssAsyncChunkLoadFailed =
+      error?.message?.includes('CSS chunk') ?? false;
```

Comment:
`Boolean()` should generate less code than `?? false`.

```suggestion
      Boolean(error?.message?.includes('CSS chunk'))
```